ils_middleware/tasks/alma/post_bfinstance.py

```diff
@@ -11,6 +11,12 @@
 logger = logging.getLogger(__name__)
 
 
+def get_env_vars(institution):
+    uri_region = Variable.get(f"alma_uri_region_{institution}")
```

Comment:
Just to confirm; we'll need to create two variables for each institution through the Airflow UI?

Reply:
yes, that is correct.